mlflow/pytorch/__init__.py

```diff
@@ -59,6 +59,9 @@
 def log_model(pytorch_model, artifact_path, conda_env=None, code_paths=None,
               pickle_module=mlflow_pytorch_pickle_module, **kwargs):
     """
+    log_model(pytorch_model, artifact_path, conda_env=None, code_paths=None,\
```

Comment:
If an alternate method prototype is included in the first line of the docstring, Sphinx will use it in place of the autogenerated prototype (see http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_docstring_signature and https://stackoverflow.com/questions/43979369/sphinx-autodoc-with-parameter-from-a-mocked-module-on-readthedocs).

Reply:
good to know!